Source/TheSpacePort/Parking.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TheSpacePort
+{
+    public class Parking
```

Comment:
Tänker att ni kanske borde hantera själva "parkeringen" separat och att den i sin tur tar in parkeringsplatsobjekt?

Reply:
Det är planen